handlers/account.go

```diff
@@ -72,7 +69,7 @@ func (h *AccountHandler) Activate(w http.ResponseWriter, r *http.Request) error
 		return NewHandlerError(err, "wrong token kind", http.StatusForbidden).WithMessage("wrong token kind was used")
 	}
 
-	a, err := h.accountService.Activate(auth.ID, request.Longtitude, request.Latitude, request.TZName)
+	a, err := h.accountService.Activate(auth.ID, request.Longitude, request.Latitude, request.TZName)
```

Comment:
Latitude and Longitude were stored with the building, which was originally created in the db at the time of account activation. With the new EnergyQuery facility, however, lat and lon are no longer in the database as separate fields. I expect that, if a campaign needs to know the building's timezone, this data will be have to be uploaded as part of the response to an EnergyQuery (e.g. the BuildingProfile EnergyQuery seems particularly well suited for this). 

So.. perhaps skip these parameters here?

One other thing to look out for: usage of ISO8601 fully qualified time stamps in response values of the API. In the v2 API, we already started accepting unix time timestamps in uploads and measurements.

In some API responses, however, we always used ISO8601 fully qualified time (including timezone offset) as timestamps. This was based on the assumption that all things related to accounts, via building(s) could always look at the timezone associated with the building and then onstruct a fully qualified ISO8601 timestamp, even if only a unix timewould have been available. When we ditch the manadatory TZName, this assumption no longer holds.

As a consequence, in the API v3 we may now fully switch to unix time for timestamps in the API. Any process that requires knowing or displaying a local time should get the TZName separately (analysis: from an uploaded EnergyQuery; NeedForHeat GearUp app (e.g. when displaying timestamps in the datasouces list: based on the timezone stored in the smartphone). 

In particular, for the API this means:
- any timestamp in the response now also should be specified in unix time only. 
- it may be wise to stop accepting ISO8601 timestamps in requests, only support Unix times.

Reply:
Seems to cause no issues? Of course we'd need to test this when this goes live on testing, but looking into the code there were only a few instances of `.toISOString()` which I changed to `.toLocaleString()` which (according to chatgpt) should use the device's timezone to convert.

I've done this change which was quick and easy but this would likely mean that WeatherInterpolationLocation should be done first before anything else in the DataSourceList most likely.

Reply:
I only see .toLocaleString() in the Swagger .js files. 
In the JSON requests and responses themselves, only Unix time should be supported in API v3. 
I am not sure that .toLocateString() achieves what we want. 

Shoudn't you use something like:

 unixTimeInSeconds = Math.floor(someDateTime.getTime() / 1000);

Nevertheless, I will approve the PR, so you can check.